src/projection.ts

```diff
@@ -5,29 +5,51 @@ export type ProjectionType = VgProjectionType;
 
 export interface Projection {
   /**
-   * The type of the projection. The default is "mercator".
+   * The cartographic projection to use. This value is case-insensitive, for example `"albers"` and `"Albers"` indicate the same projection type. You can find all valid projection types [in the documentation](https://vega.github.io/vega-lite/docs/projection.html#projection-types).
+   *
+   * __Default value:__ `mercator`
    */
   type?: ProjectionType;
+
   /**
-   * The clip angle of the projection. The default is `null`.
+   * Sets the projection’s clipping circle radius to the specified angle in degrees. If `null`, switches to [antimeridian](http://bl.ocks.org/mbostock/3788999) cutting rather than small-circle clipping.
    */
   clipAngle?: number;
+
   /**
-   * Sets the projection’s viewport clip extent to the specified bounds in pixels. The default is `null`.
+   * Sets the projection’s viewport clip extent to the specified bounds in pixels. The extent bounds are specified as an array `[[x0, y0], [x1, y1]]`, where `x0` is the left-side of the viewport, `y0` is the top, `x1` is the right and `y1` is the bottom. If `null`, no viewport clipping is performed.
    */
   clipExtent?: number[][];
+
   /**
-   * The center of the projection. The default is `[0, 0]`.
+   * Sets the projection’s center to the specified center, a two-element array of longitude and latitude in degrees.
+   *
+   * __Default value:__ `[0, 0]`
    */
   center?: number[];
+
   /**
-   * The rotation of the projection. The default is `[0, 0, 0]`.
+   * Sets the projection’s three-axis rotation to the specified angles, which must be a two- or three-element array of numbers [`lambda`, `phi`, `gamma`] specifying the rotation angles in degrees about each spherical axis. (These correspond to yaw, pitch and roll.)
+   *
+   * __Default value:__ `[0, 0, 0]`
    */
   rotate?: number[];
+
   /**
-   * The desired precision of the projection. If precision is not specified, returns the projection’s current resampling precision which defaults to √0.5 ≅ 0.70710….
+   * Sets the threshold for the projection’s [adaptive resampling](http://bl.ocks.org/mbostock/3795544) to the specified value in pixels. This value corresponds to the [Douglas–Peucker distance](http://en.wikipedia.org/wiki/Ramer%E2%80%93Douglas%E2%80%93Peucker_algorithm). If precision is not specified, returns the projection’s current resampling precision which defaults to `√0.5 ≅ 0.70710…`.
    */
   precision?: String;
+
+  /** The following properties are all supported for specific types of projections. Consult the d3-geo-projection library for more information: https://github.com/d3/d3-geo-projection */
```

Comment:
If you want do it a lazy way like this, use `/*` not `/**` otherwise this comment will be attached to only `coefficient`, which is not what you want.